control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller_test.go

```diff
@@ -693,8 +694,7 @@ func TestEtcdRestoredCondition(t *testing.T) {
 	}
 }
 
-func TestEventHandling(t *testing.T) {
-	t.Parallel()
+func sampleHCP() (*hyperv1.HostedControlPlane, error) {
```

Comment:
is there a reason to return an error? should we just panic?

Reply:
Updated it to take a testing.T and just Fatal if it fails